demo/api/schema.gql

```diff
@@ -460,7 +460,6 @@ input NewsUpdateInput {
   image: DamImageBlockInput
   slug: String
   title: String
-  visible: Boolean
```

Comment:
This should not be changed in this PR

Reply:
https://github.com/vivid-planet/comet/pull/1145